### PR TITLE
cet release notes

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/continuous-error-tracking.md [hunk 1]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
-
 This release does not include any early access features.
 
 ```mdx-code-block
   </TabItem>
   <TabItem value="Fixed issues">
 ```
-This release does not include any fixed issues.
+
+- The Events Summary page is taking longer (exceeding over eight hours) to display the events. (CET-1356)
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:40]:
```suggestion
- The Events Summary page is taking longer (exceeding eight hours) to display the events. (CET-1356)  
```

File: release-notes/continuous-error-tracking.md [hunk 2]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
+
+#### May 08, 2023, version 79111
+
+Harness Continuous Error Tracking (CET) is now available for public preview to provide developer-first observability for modern applications. 
+
+CET helps developers proactively identify and resolve errors in modern applications. As the latest module in the Harness Platform, CET helps developers proactively identify and resolve errors across the entire software delivery lifecycle (SDLC).
+
+Here are CET’s key features:
+
+* Exception/error event summary: displays a comprehensive list of all the exceptions and events that occur in your.
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:84]:
```suggestion
* Exception/error event summary: Displays a comprehensive list of all the exceptions and events that occur in your.
```

Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:84]: You may be missing a word after "your" on line 84

File: release-notes/continuous-error-tracking.md [hunk 3]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
+
+* Event explorer: This helps you explore the events list in detail.
+
+* Automated Root Cause Analysis (ARC) Screen: Provides a powerful mechanism to view every exception’s source code, variables, and environment state.
+
+* Hidden, resolved, and resurfaced events: helps you manage and troubleshoot the events and exceptions systematically by providing you with the ability to mark them as hidden, resolved, or resurfaced.
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:90]:
```suggestion
* Hidden, resolved, and resurfaced events: Helps you manage and troubleshoot the events and exceptions systematically by providing you with the ability to mark them as hidden, resolved, or resurfaced.
```

File: release-notes/continuous-error-tracking.md [hunk 4]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
+
+CET helps developers proactively identify and resolve errors in modern applications. As the latest module in the Harness Platform, CET helps developers proactively identify and resolve errors across the entire software delivery lifecycle (SDLC).
+
+Here are CET’s key features:
+
+* Exception/error event summary: displays a comprehensive list of all the exceptions and events that occur in your.
+
+* Event explorer: This helps you explore the events list in detail.
+
+* Automated Root Cause Analysis (ARC) Screen: Provides a powerful mechanism to view every exception’s source code, variables, and environment state.
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:88]:
```suggestion
* Automated Root Cause Analysis (ARC) screen: Provides a powerful mechanism to view every exception’s source code, variables, and environment state.
```

File: release-notes/continuous-error-tracking.md [hunk 5]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
+
+* Hidden, resolved, and resurfaced events: helps you manage and troubleshoot the events and exceptions systematically by providing you with the ability to mark them as hidden, resolved, or resurfaced.
+
+* Critical events: You can add a condition that specifies when an event should be marked as critical.
+
+* Source Attach: provides you with the ability to see the decompiled source code for an event on the ARC screen by connecting your GitLab and GitHub source repositories to CET.
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:94]:
```suggestion
* Source Attach: Provides you with the ability to see the decompiled source code for an event on the ARC screen by connecting your GitLab and GitHub source repositories to CET.
```

File: release-notes/continuous-error-tracking.md [hunk 6]
```diff
@@ -18,40 +18,83 @@ Review the notes below for details about recent changes to Continuous Error Trac
...
+
+* Shift Left with Harness CI integration: CET natively integrates into Harness Continuous Integration so that you can gain a more complete picture of exceptions.
+
+* JIRA integration: provides you with the ability to create and view Jira tickets for a specific event.
```
Comment (brian-f-harness) [inline release-notes/continuous-error-tracking.md:98]:
```suggestion
* Jira integration: Provides you with the ability to create and view Jira tickets for a specific event.
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Added a couple minor suggestions and noted one question.

**Comment by SudheendraKatte** [comment]
Tech review and edit review complete. Fixed minor text issues.

File: release-notes/continuous-error-tracking.md (hunk 1 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

File: release-notes/continuous-error-tracking.md (hunk 2 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

File: release-notes/continuous-error-tracking.md (hunk 3 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

File: release-notes/continuous-error-tracking.md (hunk 4 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

File: release-notes/continuous-error-tracking.md (hunk 5 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

File: release-notes/continuous-error-tracking.md (hunk 6 above)
Comment (SudheendraKatte) [reply release-notes/continuous-error-tracking.md]: Done

**Pull request merged**